lib/rasputin.rb

```diff
@@ -21,7 +21,7 @@ class Engine < ::Rails::Engine
 
     config.assets.configure do |env|
       silence_warning = !Sprockets::VERSION.start_with?("4")
-      env.register_preprocessor 'application/javascript', Rasputin::RequirePreprocessor, silence_deprecation: silence_warning
+      env.register_preprocessor 'application/javascript', Rasputin::RequirePreprocessor
```

Comment:
I'm wondering the reasoning of not silencing the warnings in this case.

Reply:
Looks like only `register_engine` supports that option.    I originally added the `silence_deprecation` option to the `register_preprocessor` call but it threw the following exception:

```undefined method `call' for {:silence_deprecation=>true}:Hash```

I also tried using the same `SILENCE_DEPRECATIONS` approach we added to EMEX but it appears the Tilt::Template warnings are handled outside of `ActiveSupport`.  Since there was only one remaining warning I figured it probably wasn't worth spending much more time on.  Hopefully we can drop support for the ember code soon.